src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -429,19 +423,14 @@ applyUpgrades v parentHeader (BlockCreationTime currCreationTime) =
       initCapabilities [mkMagicCapSlot "REMEDIATE"]
 
     applyTx tx = do
-
       infoLog $ "Running upgrade tx " <> sshow (_cmdHash tx)
```

Comment:
what's up with the whitespace deletions here, please undo

Reply:
It helped me reading the code. Distinguishing which lines belong to the same function.

Reply:
I can put back in but the newlines here seem excessive

Reply:
You can leave them in but going forward avoid needless/opinionated changes please